### PR TITLE
Update add-another-email-alias-for-a-user.md

### LINKED ISSUE #12479: Unhelpful instructions for managing via AD
**Did you get "This user is synchronized with your local Active Directory. Some details can be edited only through your local Active Directory" message?**
_If you get this error message it means that you need to add the alias in your on-premises Active Directory._

No shit.  That's what I came to this document for in the first place.  This has to be the singularly most unhelpful KB I've seen so far.
Where are the instructions on _how_ to edit it in AD?

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 94754a91-ac3b-9b1c-ded9-3fdab2dd229c
* Version Independent ID: 517f9726-1861-8b1c-fe9d-6c50047e16d3
* Content: [Add another email alias for a user - Microsoft 365 admin](https://learn.microsoft.com/en-us/microsoft-365/admin/email/add-another-email-alias-for-a-user?view=o365-worldwide)
* Content Source: [microsoft-365/admin/email/add-another-email-alias-for-a-user.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/admin/email/add-another-email-alias-for-a-user.md)
* Service: **microsoft-365-business**
* GitHub Login: @kwekuako
* Microsoft Alias: **kwekua**

### PR DESCRIPTION
**Pull request by msbemba**
Added information on how to edit the alias in AD

Fixes https://github.com/MicrosoftDocs/microsoft-365-docs/issues/12479

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/admin/email/add-another-email-alias-for-a-user.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ If you get this error message it means that it's taking a bit longer to finish s
 
 ### Did you get "This user is synchronized with your local Active Directory. Some details can be edited only through your local Active Directory" message?
 
-If you get this error message it means that you need to add the alias in your on-premises Active Directory.
+If you get this error message it means that you need to add the alias in your on-premises Active Directory. Open **Active Directory** > **Right Click on the user account you would like to Edit** > **Properties** > **Attribute Editor** > **Proxyaddresses** > **Add the desired alias** and wait for it to sync to Azure Active directory.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/email/add-another-email-alias-for-a-user.md]:
```suggestion
If you get this error message it means that you need to add the alias in your on-premises Active Directory. Open Azure Active Directory and select the user account you would like to edit. Select **Properties** > **Attribute Editor** > **Proxyaddresses** and add the required alias, then wait for it to sync to Azure Active directory.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@kwekuako Please merge this PR. Thanks

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kwekuako Gentle reminder

**Pull request merged**